### ISSUE TITLE
Ionic cordova platforms add [ios/android] type error

### ISSUE THREAD

**Issue by MarioC3** [ionic-team/ionic-cli#4615]
**Description:**
Whenever I want to add a platform (ios or android) using `ionic cordova platforms add ` I receive the following error:
```
Failed to fetch platform cordova-ios@^6.1.0
Probably this is either a connection problem, or platform spec is incorrect.
Check your connection and platform name/version/URL.
CordovaError: TypeError: Cannot assign to read only property 'valueOf' of object '[object Object]'
```
When I try the command using only `cordova platforms add` (without the ionic cli) the platform can be fetched and it can be generated.

**Steps to Reproduce:**
Trying to add a cordova platform using the ionic cli:
`ionic cordova platforms add`

**Output:**
```
> cordova platform add ios
Using cordova-fetch for cordova-ios@^6.1.0
Failed to fetch platform cordova-ios@^6.1.0
Probably this is either a connection problem, or platform spec is incorrect.
Check your connection and platform name/version/URL.
CordovaError: TypeError: Cannot assign to read only property 'valueOf' of object '[object Object]'
[ERROR] An error occurred while running subprocess cordova.
        
        cordova platform add ios exited with exit code 1.
        
        Re-running this command with the --verbose flag may provide more information.
```

**My `ionic info`:**

```
Ionic:

   Ionic CLI                     : 6.12.2 (/Users/marioacarlos/.nvm/versions/node/v12.16.2/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.5.0
   @angular-devkit/build-angular : 0.901.12
   @angular-devkit/schematics    : 10.2.0
   @angular/cli                  : 9.1.12
   @ionic/angular-toolkit        : 2.3.3

Cordova:

   Cordova CLI       : 10.0.0
   Cordova Platforms : none
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 5.0.0, (and 19 other plugins)

Utility:

   cordova-res (update available: 0.15.2) : 0.14.0
   native-run (update available: 1.2.2)   : 1.0.0

System:

   Android SDK Tools : 26.1.1 (/Users/marioacarlos/Library/Android/sdk)
   ios-deploy        : 1.10.0
   ios-sim           : ios-sim/9.0.0 darwin-x64 node-v12.16.2
   NodeJS            : v12.16.2 (/Users/marioacarlos/.nvm/versions/node/v12.16.2/bin/node)
   npm               : 6.14.8
   OS                : macOS Big Sur
   Xcode             : Xcode 12.2 Build version 12B45b
```

**Comment by ath0mas** [comment]
Fixed, see apache/cordova-cli#543

**Comment by MarioC3** [comment]
Thank you!
